debug_mode: true
enable_partial_update: true
enable_mocap_switch: false
mahalanobis_d_sonar: 100
mahalanobis_d_rgbd_velocity: 120000
mahalanobis_d_mocap_z: 20
mahalanobis_d_mocap_velocity: 50
enable_measurements: true
mocap_override_channel: 6
